parser-typechecker/src/Unison/Runtime/Machine.hs

```diff
@@ -388,17 +393,36 @@ exec !env !denv !_activeThreads !ustk !bstk !k _ (BPrim1 DBTX i)
           bstk <- bump bstk
           bstk <$ pokeBi bstk (Util.Text.pack tx)
       pure (denv, ustk, bstk, k)
+exec !env !denv !_activeThreads !ustk !bstk !k _ (BPrim1 SDBL i) = do
+  tl <- peekOffBi bstk i
+  bstk <- bump bstk
+  pokeS bstk . encodeSandboxListResult =<< sandboxList env tl
+  pure (denv, ustk, bstk, k)
 exec !_ !denv !_activeThreads !ustk !bstk !k _ (BPrim1 op i) = do
   (ustk, bstk) <- bprim1 ustk bstk op i
   pure (denv, ustk, bstk, k)
-exec !env !denv !_activeThreads !ustk !bstk !k _ (BPrim2 SDBX i j) = do
-  s <- peekOffS bstk i
-  c <- peekOff bstk j
-  l <- decodeSandboxArgument s
-  b <- checkSandboxing env l c
-  ustk <- bump ustk
-  poke ustk $ if b then 1 else 0
-  pure (denv, ustk, bstk, k)
+exec !env !denv !_activeThreads !ustk !bstk !k _ (BPrim2 SDBX i j)
+  | sandboxed env =
+      die "attempted to use sandboxed operation: sandboxLinks"
```

Comment:
Should the error message say "sandboxLinks" here or something else?

Reply:
The error message is correct, but I'd put it in the wrong case there. It was meant for `SDBL`.